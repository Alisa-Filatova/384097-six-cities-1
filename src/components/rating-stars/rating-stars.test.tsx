import * as React from 'react';
import * as renderer from 'react-test-renderer';
import RatingStars from './rating-stars';

describe('RatingStars', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(
        <RatingStars rating={4} showValue />
    ).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
