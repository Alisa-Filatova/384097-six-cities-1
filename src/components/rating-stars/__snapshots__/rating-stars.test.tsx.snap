// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingStars renders correctly 1`] = `
<div
  className="place-card__rating rating"
>
  <div
    className="place-card__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "80%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
  <span
    className="place-card__rating-value rating__value"
  >
    4
  </span>
</div>
`;
